.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "app/**/*.js": ["prettier --write", "eslint --fix"]
```

Comment:
Hi @corners2wall, This will only lint files inside app, we want it on entire project

Reply:
changed